vbox_create.sh

```diff
@@ -176,13 +191,32 @@ function create_cluster_VMs {
   VBN1="${bootstrap_interfaces[1]?Need a Virtualbox network 2 for the bootstrap}"
   VBN2="${bootstrap_interfaces[2]?Need a Virtualbox network 3 for the bootstrap}"
 
-  #
-  # Add the ipxe USB key to the vbox storage registry as an immutable
-  # disk, so we can share it between several VMs.
-  #
-  IPXE_DISK=$P/ipxe.vdi
-  cp files/default/ipxe.vdi $IPXE_DISK
-  $VBM modifyhd -type immutable $IPXE_DISK
+  if [[ $CLUSTER_VM_EFI == true ]]; then
```

Comment:
TIL that the ipxe disk "re-registration" is failing because its uuid is already  registered with virtualbox.  Regenerating the UUID removes the need for this

```
vboxmanage internalcommands sethduuid files/default/ipxe.vdi
```